cfgov/jinja2/v1/sublanding-page-1/index.html

```diff
@@ -33,22 +19,46 @@
         - Sidebar with latest activities
 #}
 
+{% block pre_content %}
+    {% set settings = {
+        'heading':          'Sample Header(H3)',
+        'hide_header_slug': true,
+        'image': {
+            'url':  '/static/img/clipboard-round-300x300.png',
+            'round' : true
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/0152b8b34825bacf002abfe0394409cf6e87bba3